blccpsapibukkit/pom.xml

```diff
@@ -65,7 +65,7 @@
         <dependency>
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
-            <version>2.6.2</version>
+            <version>2.8.6</version>
```

Comment:
Thanks for bumping. I guess we can even set this dependency to `provided` as it is in bukkit already

Reply:
Right, that's something we could do.

Just a side note: Since this project supports multiple versions of Bukkit, it would be better to compile and relocate this dependency for compatibility reasons. This would ensure that a new Gson version (which could be shipped with a new version of the Minecraft vanilla server) - or even a modified version of Mojang - wouldn't cause conflicts due to breaking changes. Since Gson is only used to a very small and simple extent here, and it's unlikely that the Gson API used in this project will change in the near future, this risk can be neglected for now.